src/test/suite/editingTests.test.ts

```diff
@@ -0,0 +1,58 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from "assert";
+import { withTestNotebook } from "./helper";
+import {
+  EventEmitter,
+  NotebookCellKind,
+  NotebookConcatTextDocument,
+  NotebookDocument,
+  Uri,
+} from "vscode";
+import { EnhancedNotebookConcatTextDocument } from "../../nativeNotebookConcatTextDocument";
+import { IVSCodeNotebook } from "../../common/types";
+
+suite("Editing Tests", () => {
+  test("Edit a notebook", () => {
+    withTestNotebook(
```

Comment:
I think we need a lot more tests to verify this is working correctly. Entered an issue for that:
https://github.com/microsoft/vscode-jupyter/issues/7175